src/cljc/imcljs/internal/defaults.cljc

```diff
@@ -40,6 +40,12 @@
                                   (missing? options :format) (assoc :format "json")))
     request))
 
+(defn wrap-put-defaults [request options]
```

Comment:
it would be nice to add a docstring here, even if it's terse - I can see this is a wrapper function but we could describe what it wraps or what it is used for/by :)